src/app/views/response/feedback.py

```diff
@@ -0,0 +1,19 @@
+from typing import Optional
+
+from pydantic import BaseModel
+
+
+class Feedback(BaseModel):
+    player_name: str
+    vote: int
+    prediction: str
+    confidence: Optional[float]
+    feedback_text: Optional[str]
+    proposed_label: Optional[str]
+
+
+class FeedbackCount(BaseModel):
+    count: int
+    possible_ban: bool
+    confirmed_ban: bool
+    confirmed_player: bool
```

Comment:
add vote to this

Reply:
TODO

Reply:
> add vote to this

count is the summation of votes see
https://github.com/Bot-detector/public-api/blob/4c1180aa78f47297df01280f7c21c719b9b79d35/src/app/models/feedback.py#L79